.github/workflows/pytest.yml

```diff
@@ -8,6 +8,9 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
+  workflow_dispatch:
+  schedule:
+    - cron: "0 5 * * *"
```

Comment:
Is this a good reason to have this cron job for the pytest?  In the current state, pre this PR, Everytime homeassistant updates, we update this package automatically which tests using pytest. I'm a bit wary of getting pinged with failures more than necessary with this additional cron job.

Reply:
I personally find it useful to run a cron weekly so that any breaking change of 3rd party things are detected seperately and before I make a real change. I can remove the cron when you prefer that

Reply:
I can always remove it if I get annoyed.  Thanks